scripts/gcp_setup/setup_forseti.py

```diff
@@ -31,8 +31,14 @@ def run():
     parser.add_argument('--no-iam-check',
                         action='store_true',
                         help='Bypass IAM check for user running script')
-    parser.add_argument('--branch',
-                        help='Which Forseti branch to deploy')
+    parser.add_argument('--config',
```

Comment:
What do you think about adding a --basic and/or --advanced flag. Where when it is in basic mode, or not advanced, then it uses default answers as much as possible? So that the total number of questions is reduced to the minimal required?